querybook/server/logic/schedule.py

```diff
@@ -65,7 +65,7 @@ def create_task_schedule(
         kwargs=kwargs or {},
         options=options or {},
         enabled=enabled,
-        task_type=get_schedule_task_type(name),
+        task_type=get_schedule_task_type(name).value,
```

Comment:
@adamstruck thanks for fixing this! could you also update here https://github.com/pinterest/querybook/blob/21c3fd4aa75ef7437792fa735fbdce3641620333/querybook/server/logic/demo.py#L75